custom_components/miele/translations/de.json

```diff
@@ -417,10 +416,10 @@
                             "dust_box_missing": "Fehlender Staubbehälter",
                             "blocked_drive_wheels": "Antriebsrad blockiert",
                             "blocked_brushes": "Bürsten blockiert",
-                            "check_dust_box_and_filter": "Checke Staubbehälter und Filter",
+                            "check_dust_box_and_filter": "Staubbehälter und Filter prüfen",
                             "internal_fault_reboot": "Interne Fehler - Neustart",
                             "blocked_front_wheel": "Vorderrad blockiert",
```

Comment:
According to the error message description in the Scout RX3 Home Vision manual: `Vorderrad verschmutzt` (`Front wheel dirty` in the English version).